.env.example

```
@@ -5,6 +5,10 @@ APP_ENV=development
 # APP_CONFIG_FILE=./config.production.json
 
 # 敏感信息建议通过环境变量覆盖，不写入配置文件
+DATABASE_HOST=localhost
+DATABASE_PORT=5432
+DATABASE_USER=admin
+DATABASE_NAME=mydb
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABASE_NAME key should go before the DATABASE_PORT key

(UnorderedKey)